packages/rn-async-storage-flipper/index.js

```diff
@@ -17,26 +17,56 @@ function ReactNativeAsyncStorageFlipper(storage) {
     bootstrapPlugin()
         .then((currentConnection) => {
             if (currentConnection) {
-                storage.getAllKeys().then((keys) => {
-                    storage.multiGet(keys).then((data) => {
-                        data.map((_, i, store) => {
-                            let key = store[i][0];
-                            let value = store[i][1];
-                            let needsToBeParsed =
-                                (value.startsWith('{') &&
-                                    value.endsWith('}')) ||
-                                (value.startsWith('[') && value.endsWith(']'));
-                            if (needsToBeParsed) {
-                                value = JSON.parse(value);
-                            }
-                            currentConnection.send('newElement', {
-                                key,
-                                value,
-                                id: key,
-                            });
-                        });
+              const setItem = storage.setItem;
```

Comment:
Please use the project's indentation of 4 spaces.

Reply:
Fixed